tests/integration_tests/network.py

```diff
@@ -0,0 +1,174 @@
+from pystarport import ports
+from web3.middleware import geth_poa_middleware
+
+from .cosmoscli import CosmosCLI
+from .utils import supervisorctl, wait_for_port
+
+DEFAULT_CHAIN_BINARY = "evmosd"
+
+
+class Evmos:
+    def __init__(self, base_dir, chain_binary=DEFAULT_CHAIN_BINARY):
+        self._w3 = None
+        self.base_dir = base_dir
+        self.config = json.loads((base_dir / "config.json").read_text())
+        self.enable_auto_deployment = False
+        self._use_websockets = False
+        self.chain_binary = chain_binary
+
+    def copy(self):
+        return Evmos(self.base_dir)
+
+    @property
+    def w3_http_endpoint(self, i=0):
+        port = ports.evmrpc_port(self.base_port(i))
+        return f"http://localhost:{port}"
+
+    @property
+    def w3_ws_endpoint(self, i=0):
+        port = ports.evmrpc_ws_port(self.base_port(i))
+        return f"ws://localhost:{port}"
+
+    @property
+    def w3(self, i=0):
+        if self._w3 is None:
+            if self._use_websockets:
+                self._w3 = web3.Web3(
+                    web3.providers.WebsocketProvider(self.w3_ws_endpoint)
+                )
+            else:
+                self._w3 = web3.Web3(web3.providers.HTTPProvider(self.w3_http_endpoint))
+        return self._w3
+
+    def base_port(self, i):
+        return self.config["validators"][i]["base_port"]
+
+    def node_rpc(self, i):
+        return "tcp://127.0.0.1:%d" % ports.rpc_port(self.base_port(i))
+
+    def use_websocket(self, use=True):
+        self._w3 = None
+        self._use_websockets = use
+
+    def cosmos_cli(self, i=0):
+        return CosmosCLI(
+            self.base_dir / f"node{i}", self.node_rpc(i), self.chain_binary
+        )
+
+    def node_home(self, i=0):
+        return self.base_dir / f"node{i}"
+
+    def supervisorctl(self, *args):
+        return supervisorctl(self.base_dir / "../tasks.ini", *args)
+
+
+# another Cosmos chain to be used on IBC transactions
+class Chainmain:
+    def __init__(self, base_dir):
+        self.base_dir = base_dir
+        self.config = json.loads((base_dir / "config.json").read_text())
+
+    def base_port(self, i):
+        return self.config["validators"][i]["base_port"]
+
+    def node_rpc(self, i):
+        return "tcp://127.0.0.1:%d" % ports.rpc_port(self.base_port(i))
+
+    def cosmos_cli(self, i=0):
+        return CosmosCLI(self.base_dir / f"node{i}", self.node_rpc(i), "chain-maind")
+
+
+#  Hermes IBC relayer
+class Hermes:
+    def __init__(self, config: Path):
+        self.configpath = config
+        self.config = tomlkit.loads(config.read_text())
+        self.port = 3000
+
+
+class Geth:
+    def __init__(self, w3):
+        self.w3 = w3
+
+
+def setup_evmos(path, base_port, long_timeout_commit=False):
+    cfg = Path(__file__).parent / (
+        "configs/default.jsonnet"
+        if long_timeout_commit
+        else "configs/long_timeout_commit.jsonnet"
+    )
+    yield from setup_custom_evmos(path, base_port, cfg)
```

Comment:
this is the wrong way around, no? I would expect cfg to be `long_timeout_commit.jsonnet` if the boolean value is true and not `else`

```suggestion
    cfg = Path(__file__).parent / (
        "configs/long_timeout_commit.jsonnet"
        if long_timeout_commit
        else "configs/default.jsonnet"
    )
```